src/components/ColorPicker.tsx

```diff
@@ -1,20 +1,29 @@
 import AppStore from '../stores/AppStore';
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
 import { Popover, IconButton } from '@material-ui/core';
 import { SketchPicker } from 'react-color';
 import { changeCourseColor, changeCustomEventColor } from '../actions/AppStoreActions';
 import { ColorLens } from '@material-ui/icons';
 import ReactGA from 'react-ga';
 import analyticsEnum, { logAnalytics } from '../analytics';
 
-class ColorPicker extends PureComponent {
+interface ColorPickerProps {
+    color: string
+    analyticsCategory: string
+    isCustomEvent: boolean
+    customEventID?: number // present if isCustomEvent is true
+    //present if isCustomEvent is false:
```

Comment:
Shouldn't this comment be after the next line?

Reply:
I guess these comments were a bit unclear. Updated in https://github.com/icssc/AntAlmanac/pull/444/commits/c5d21ef8a437def33d363f8b34190dad5d124123